pyaedt/modules/Material.py

```diff
@@ -272,579 +492,507 @@ def desktop(self):
+            return False
 
-        Returns
-        -------
 
-        """
-        if fullname in self.property:
-            try:
-                return float(self.property[fullname].property_value[0].value)
-            except:
-                ds = self.property[fullname].property_value[0].dataset
-                if ds:
-                    try:
-                        return float(ds.ds[0][1])
-                    except:
-                        return 0
-                return 0
+class Material(CommonMaterial, object):
+    """Class for Material Properties"""
 
-    @aedt_exception_handler
-    def get_data(self, fullname):
-        """
+    def __init__(self, parent, name, props=None):
+        CommonMaterial.__init__(self, parent, name, props)
+        self.thermal_material_type = "Solid"
+        if "thermal_material_type" in self._props:
+            self.thermal_material_type = self._props["thermal_material_type"]["Choice"]
+        if "PhysicsTypes" in self._props:
+            self.physics_type = self._props["PhysicsTypes"]["set"]
+        else:
+            self.physics_type = ['Electromagnetic', 'Thermal', 'Structural']
+            self._props["PhysicsTypes"]=OrderedDict({"set":['Electromagnetic', 'Thermal', 'Structural']})
+
+        for property in MatProperties.aedtname:
+            if property in self._props:
+                mods = None
+                if "ModifierData" in self._props:
+                    for mod in self._props["ModifierData"]["ThermalModifierData"]["all_thermal_modifiers"]:
+                        if isinstance(self._props["ModifierData"]["ThermalModifierData"]["all_thermal_modifiers"][mod],
+                                      list):
+                            for one_tm in self._props["ModifierData"]["ThermalModifierData"]["all_thermal_modifiers"][
+                                mod]:
+                                if one_tm["Property:"] == property:
+                                    if mods:
+                                        mods = [mods]
+                                        mods.append(one_tm)
+                                    else:
+                                        mods = one_tm
+                        else:
+                            if self._props["ModifierData"]["ThermalModifierData"]["all_thermal_modifiers"][mod][
+                                "Property:"] == property:
+                                mods = self._props["ModifierData"]["ThermalModifierData"]["all_thermal_modifiers"][mod]
+                self.__dict__["_" + property] = MatProperty(self, property, self._props[property], mods)
+            else:
+                self.__dict__["_" + property] = MatProperty(self, property,
+                                                            MatProperties.get_defaultvalue(aedtname=property), None)
+        pass
 
-        Parameters
-        ----------
-        fullname :
-            
+    @property
+    def permittivity(self):
+        """
```

Comment:
Docstrings for properties must have a one line short description.  Copy/paste what we have in `Returns` should be sufficient.  Reasoning is when we finally merge #176, we don't want to have empty descriptions in our tables.  It's not too obvious that our descriptions are empty, but it will be quite obvious when each property/method stands alone.